Snake oil be snake oiling (#7254)
jenkins-infra/jenkins.io: content/security/reporting.adoc

  This behavior is commonly the case in plugins integrating with a hosted service (e.g., some connection tests) and while permissions beyond Overall/Read should be required to cause Jenkins to send a request, the impact is negligible.
* Vulnerabilities in dependencies without a plausible or demonstrated exploit will not be treated as vulnerabilities.
  While we may inform maintainers about the need to update their dependencies and track progress in the SECURITY Jira project, no security advisory will be published for these.

> * Claims of malware in plugin:durable-task[Durable Task] plugin or https://github.com/jenkinsci/lib-durable-task[lib-durable-task] unless substantiated (e.g., local builds from source are unaffected).

Reviewer: ```suggestion
* Claims of malware in the plugin:durable-task[Durable Task] plugin or https://github.com/jenkinsci/lib-durable-task[lib-durable-task] unless substantiated (for example, local builds from source are unaffected).
```
A small suggestion to include `the` for better grammar and adjusting the language to use `for example` [instead of `e.g.`](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg).

Author: FWIW https://github.com/jenkins-infra/jenkins.io/blob/master/STYLEGUIDE.adoc#assorted-comments is the more immediately relevant reference for the latter.

Re the former, the current usage is consistent with most other occurrences at least in `/security/`. Is this new?

Reviewer: Thanks @daniel-beck, I forget that it's listed there half the time.

As for the former, I thought since it is part of the Non-issues list it would be more consistent with that writing. That list is using articles throughout and does not necessarily have the same usage for plugins as what I found in the Advisories. However, if that wasn't the correct place to use as a reference, would you be able to share a link to what you're referring to?

Author: > That list is using articles throughout and does not necessarily have the same usage for plugins as what I found in the Advisories.

Unsure what you're referring to here. Isn't "the plugin"/"a plugin" grammatically far from "Git plugin"? Like "a/the person" and "Daniel"? This list mentions no other specific plugin I think?

> However, if that wasn't the correct place to use as a reference, would you be able to share a link to what you're referring to?

I did a quick search for ` plugin:[a-zA-Z0-9_-]+` (160 results) and `the plugin:[a-zA-Z0-9_-]+` (12 results) in `content/security`, which indicates it's used more commonly without article, at least in advisories.

Reviewer 2: For what it’s worth, as a native speaker I usually refer to either “Timestamper” or “Email Extension” without an article and the word “plugin” (as a proper name). If using the word “plugin”, I would write “the Timestamper plugin” or “the Email Extension plugin” in much the same way as I’d write “the Lawrence J. Ellison Institute for Transformative Medicine”.

Reviewer: I think @basil explained it better than I did. In the other areas where plugins are named, it looks like they are specific versions in the advisories and almost exclusively start sentences where the article may not be necessary. Alternatively, plugin could be removed from the sentence and would fit the other usage in `/security/`.

```suggestion
* Claims of malware in plugin:durable-task[Durable Task] or https://github.com/jenkinsci/lib-durable-task[lib-durable-task] unless substantiated (e.g., local builds from source are unaffected).
```

Author: Unfortunately we cannot use plugin names like Git, Jira, or Docker without a suffix if we want to prevent confusion :-/ I guess we should change a bunch of security advisories then.

Not opposed to changing things up, but I'd like this change request to be done in the form of a style guide change.

Reviewer: I think it's more of a context question than usage question. The advisories typically use a specific version and are referring to the functions/issues within the plugin, so the article is not necessarily missing in those (the majority of usage as you found).

I don't think things need to necessarily be changed for how advisories are published, but if the goal is consistency then I can submit a style guide change for further discussion/review.

Author: I think that would be useful. I don't see the difference between

> Claims of malware in Durable Task plugin are false positive

(this PR) and

> Durable Task plugin does not contain malware

(structurally the same as advisories) so if that could be fleshed out, that would be nice.

Reviewer: @StackScribe would you have any insight/suggestions as to what would be best for this and a potential style guide entry?

Author: Two more reports since I filed this, so I won't wait to resolve this conversation. Content can always be amended later.
